front/js/productCountInput.es6

```diff
@@ -17,18 +15,16 @@
    * Subscribing on events using mediator.
    */
   function setUpListeners() {
-    mediator.subscribe('onProductsCountUp', countIncrease, triggerInputChange);
-    mediator.subscribe('onProductsCountDown', countDecrease, triggerInputChange);
+    mediator.subscribe('onProductsCountUp', countIncrease, changeSum);
+    mediator.subscribe('onProductsCountDown', countDecrease, changeSum);
 
     $(document)
-      .on('click', DOM.countUp, function countUp() {
-        mediator.publish('onProductsCountUp', $(this).siblings(DOM.countInput));
+      .on('click', OPTION.countUp, function countUp() {
+        mediator.publish('onProductsCountUp', $(this).siblings(OPTION.countInput));
       })
-      .on('click', DOM.countDown, function countDown() {
-        mediator.publish('onProductsCountDown', $(this).siblings(DOM.countInput));
+      .on('click', OPTION.countDown, function countDown() {
+        mediator.publish('onProductsCountDown', $(this).siblings(OPTION.countInput));
       });
-
-    $(DOM.countInput).on('input', productPriceSum);
```

Comment:
Why did you remove this handler?  It will break sum updating

Reply:
@artemiy312 , sorry, i mixed up this listener with countInput/countOutput. I'll return it